static/src/javascripts/projects/commercial/modules/creatives/page-skin.js

```diff
@@ -110,16 +113,12 @@ const pageSkin = (): void => {
         }
     };
 
-    const repositionSkin = (): void => {
-        const header = document.querySelector('.new-header');
-        const footer = document.querySelector('.l-footer');
-        const topBannerAd = document.querySelector('.ad-slot--top-banner-ad');
-
-        if (truskinRendered && header && topBannerAd && footer) {
-            repositionTruskin(header, footer, topBannerAd);
+    const repositionSkins = (): void => {
+        if (truskinRendered && hasPageSkin) {
+            repositionTruskin();
         }
         // This is to reposition the Page Skin to start where the navigation header ends.
-        if (!truskinRendered && hasPageSkin && isInAUEdition) {
+        if (pageskinRendered && hasPageSkin && isInAUEdition) {
```

Comment:
This is much nicer than previously where we checked if it wasn't a Truskin. Explicitly stating that we're rendering a pageskin is much nicer to read and will make way more sense down the line! 😀